src/Microsoft.Windows.CsWin32.Shared/Microsoft.Windows.CsWin32.Shared.projitems

```diff
@@ -0,0 +1,47 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MSBuildAllProjects Condition="'$(MSBuildVersion)' == '' Or '$(MSBuildVersion)' &lt; '16.0'">$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <HasSharedItems>true</HasSharedItems>
+    <SharedGUID>d820af8c-46e2-4ae4-b045-0d70b81bdf7e</SharedGUID>
+  </PropertyGroup>
+  <PropertyGroup Label="Configuration">
+    <Import_RootNamespace>Microsoft.Windows.CsWin32</Import_RootNamespace>
+  </PropertyGroup>
+  <ItemGroup>
+    <AdditionalFiles Include="$(MSBuildThisFileDirectory)AnalyzerReleases.Shipped.md" />
+    <AdditionalFiles Include="$(MSBuildThisFileDirectory)AnalyzerReleases.Unshipped.md" />
+    <AdditionalFiles Include="$(MSBuildThisFileDirectory)BannedSymbols.txt" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="$(MSBuildThisFileDirectory)ArrayTypeHandleInfo.cs" />
```

Comment:
Do we really have to give up globbing for source files here?
What if instead of using Shared Projects, we simply used globbing in the two head projects to reach into a common directory?

Reply:
IMO this limitation is an unfortunate side effect of the current recommendations for building projects. It's been a while since I tried using globs in **.projitems** and it's possible it works now.